tests/conformance_ruby/standard_filter_test.rs

```diff
@@ -808,7 +807,10 @@ fn test_date() {
         v!("July"),
         call_filter!(liquid_lib::stdlib::Date, "2006-07-05 10:00:00", "%B").unwrap()
     );
+}
 
+#[test]
+fn test_date_serialized_no_offset() {
```

Comment:
Please leave the tests as a single function.

Reply:
I figured that might be the case, so I made that a separate commit - I've backed it out.